seqr/views/utils/orm_to_json_utils.py

```diff
@@ -486,12 +479,9 @@ def get_json_for_saved_variants_with_tags(saved_variants, include_missing_varian
         tag['variantGuids'] = []
         variant_ids = variant_tag_id_map[tag_id_map[tag_guid]]
         for variant_id in variant_ids:
-            variant_guid = saved_variant_id_map.get(variant_id)
-            if variant_guid:
-                variants_by_guid[variant_guid]['tagGuids'].append(tag['tagGuid'])
-                tag['variantGuids'].append(variant_guid)
-            else:
-                missing_ids.add(variant_id)
```

Comment:
because we do an explicit filter for `filter(savedvariant_id__in=saved_variant_id_map.keys()` there is no way for the variant id to be missing, this is just very old legacy error handling